test/output-tests/gnucobol.ml

```diff
@@ -40,12 +40,28 @@ let target =
 (** [pp_relloc ppf filename] prints [filename] relative to [srcdir] if the
     latter is a directory (prefix) of [filename].  Otherwise, prints [filename]
     as a whole. *)
+    (*
 let pp_relloc =
   let srcdir_prefix = srcdir ^ Ez_file.FileOS.dir_separator_string in
   fun ppf s ->
-    match EzString.chop_prefix ~prefix:srcdir_prefix s with
-    | Some s -> Fmt.string ppf s
-    | None -> Fmt.string ppf s
+    let s =
+      match EzString.chop_prefix ~prefix:srcdir_prefix s with
+      | Some s -> s
+      | None -> s
+    in
+    Fmt.string ppf s
+*)
+
+let pp_relloc fmt s =
+  let path = EzString.split s '/' in
+  let rec iter path =
+    match path with
+    | [] -> s
+    | "import" :: "gnucobol" :: _ -> String.concat "/" path
```

Comment:
While you're at it, why not strip the `import` altogether?

Also `ppf`  stands for pretty-printing-formatter (I think). `fmt` is the old-school name, that unfortunately can be mistaken for "format-string".

Reply:
Ok for `ppf`. For `import`, I don't want to match only on `gnucobol`, as it may appear above the source directory...